### PR TITLE
feat(peer-deps): support `@angular/compiler@9`

### PR DESCRIPTION
**Pull request by fisker**
Based on #151

### CODE REVIEW

**Comment by ikatyang** [comment]
v9 is still unstable, I'll defer this PR until it's stable.

**Comment by fisker** [comment]
`v9.0.0` is released.

File: yarn.lock [hunk 1]
```diff
@@ -1478,27 +1426,12 @@ fsevents@^1.2.3:
   dependencies:
     bindings "^1.5.0"
     nan "^2.12.1"
-    node-pre-gyp "*"
```
Comment (ikatyang) [inline yarn.lock]: We should fix this line just like what we did in https://github.com/ikatyang/angular-estree-parser/pull/151#discussion_r373771788.

Comment (fisker) [reply yarn.lock]: I added it back, and resolve the conflicts.

I checked the deps of `fsevents@1.2.11`, seems `node-pre-gyp` is not listed in `dependencies` but in `bundledDependencies`.

https://unpkg.com/browse/fsevents@1.2.11/package.json

**Comment by ikatyang** [comment]
Thanks!

**Pull request merged**